client/components/web-preview/index.jsx

```diff
@@ -47,8 +56,13 @@ const WebPreview = React.createClass( {
 	getDefaultProps() {
 		return {
 			showExternal: true,
```

Comment:
Hm. We'd have to do some tooling to be able to open a preview by URL that
included any customizations. I'm not even sure that's possible with the
WP-admin customizer now since it's a POST request that's added to the
iframe.

Or do you mean just load the front page of the site without any unsaved
customizations applied?
On Mon, Mar 21, 2016 at 9:23 AM Matias Ventura <notifications@github.com>
wrote:

> In client/components/web-preview/index.jsx
> <https://github.com/Automattic/wp-calypso/pull/4136#discussion_r56820031>:
>
> > @@ -47,8 +56,13 @@ const WebPreview = React.createClass( {
> >  	getDefaultProps() {
> >  		return {
> >  			showExternal: true,
>
> I mean it allowing you to open the site if you click there.
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly or view it on GitHub
> <https://github.com/Automattic/wp-calypso/pull/4136/files/29b9995fa6b5b12855ed4742bce30bf157966b6a#r56820031>
>